{{cookiecutter.project_name}}/Makefile

```diff
@@ -83,7 +94,7 @@ rebuild-db:  ## recreates database with fixtures
 bootstrap-docker:  ## bootstrap project in docker
 	docker-compose up --build -d
 	docker-compose exec web python manage.py migrate
-	docker-compose exec web python manage.py loaddata fixtures/*
+	docker-compose exec web python manage.py loaddata fixtures/*.yaml
```

Comment:
Sorry for pointing that out but could you fix the missing quote in `docker-compose.yml` - line 15.